### ISSUE TITLE
"Bonsoir" is offensive, should be "Bonjour/Bonsoir"

### ISSUE THREAD

**Issue by TristanMarion** [riehli-r/test#1]
You are assuming it's always the evening whereas it's currently noon

**Comment by Walkoss** [comment]
@TristanMarion I agree, we're now afternoon. I suggest you submit a PR to fix this..

**Comment by riehli-r** [comment]
We sincerely apologize for offending people with this console.log of an happy evening
